models/ObjectModel.js

```diff
@@ -15,6 +15,8 @@ function ObjectModel(ip, version, protocol, objectId) {
     this.objectId = objectId;
     // The name for the object used for interfaces.
     this.name = '';
+    // The UUID used internally by Vuforia for tracking
+    this.targetId = null;
```

Comment:
This is more conceptual question why Object does not have some equivalent of publicData/privateData. Essentially, a place were data specific to object type can be stored (eg. targetId). Just some types of objects have Vuforia targets associated with them.

Reply:
Yeah, it's probably worth refactoring some of the properties of the Object data model at some point. I'm hesitating to add publicData/privateData to objects yet, as there are a set of APIs available in toolbox that let you interact with those properties (for tools and nodes), and adding the corresponding APIs for objects is beyond the scope of what I want to do in this PR. If I leave out those APIs but put targetId in publicData anyways, it feels to me that the publicData of objects will be quite different in purpose/functionality compared to that of tools/nodes, which might be confusing. I'm open to more discussion but we might need another PR for that.

Reply:
I agree that this beyond the scope here, but I wanted to raise the question.